programmers/난이도별/level01.문자열_내_마음대로_정렬하기/sjjyy.java

```diff
@@ -0,0 +1,52 @@
+package 난이도별.level01.문자열_내_마음대로_정렬하기;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class sjjyy
+{
+    public static String[] solution(String[] strings, int n)
+    {
+        int len = strings.length;
+        for (int i = 0 ; i < len ; i++)
+            strings[i] = strings[i].charAt(n) + strings[i];
+
+        Arrays.sort(strings);
+        String [] ans = new String[len];
+
+        for (int i = 0 ; i < len ; i++)
+            ans[i] = strings[i].substring(1);
+
+        return ans;
+    }
+
+    public static String[] solution2(String[] strings, int n)
+    {
+        List<String> list = new ArrayList<>();
+
+        for (String string : strings)
+            list.add(string.charAt(n) + string);
+
+        Collections.sort(list);
```

Comment:
```suggestion
        return Arrays.stream(strings)
                     .sorted((a, b) -> a.compareTo(b))
                     .sorted((a, b) -> a.charAt(n) - b.charAt(n))
                     .toArray(String[]::new);
```
이렇게 Stream의 Sorted를 이용하여 풀이하면 더 쉽습니다!